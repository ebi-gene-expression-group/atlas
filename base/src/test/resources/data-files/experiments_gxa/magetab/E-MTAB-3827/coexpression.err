Loading required package: methods
Experiment accession is: E-MTAB-3827
Reading expression data from file: /nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827/E-MTAB-3827.tsv
Coexpression matrix will be written to gzipped file: /nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827/E-MTAB-3827-coexpressions.tsv
K-means processing....done!
Coexpressions matrix will be written to: /nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827/E-MTAB-3827-coexpressions.tsv.gz
File with the same name already exists, removing it.
Comparing clusterings..............................................................................................done!
