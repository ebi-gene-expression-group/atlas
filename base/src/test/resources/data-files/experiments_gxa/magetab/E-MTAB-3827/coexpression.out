  |                                                                              |                                                                      |   0%  |                                                                              |=======                                                               |  10%.......... 
  |                                                                              |==============                                                        |  20%.......... 
  |                                                                              |=====================                                                 |  30%.......... 
  |                                                                              |============================                                          |  40%.......... 
  |                                                                              |===================================                                   |  50%.......... 
  |                                                                              |==========================================                            |  60%.......... 
  |                                                                              |=================================================                     |  70%.......... 
  |                                                                              |========================================================              |  80%.......... 
  |                                                                              |===============================================================       |  90%.......... 
  |                                                                              |======================================================================| 100%
.......... 


------------------------------------------------------------
Sender: LSF System <lsf@ebi5-188>
Subject: Job 6600988: </nfs/ma/home/atlas3-production/sw/atlasinstall_prod/atlasprod/analysis/run_coexpression_for_experiment.R E-MTAB-3827 /nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827  > in cluster <EBI> Done

Job </nfs/ma/home/atlas3-production/sw/atlasinstall_prod/atlasprod/analysis/run_coexpression_for_experiment.R E-MTAB-3827 /nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827  > was submitted from host <ebi5-258> by user <fg_atlas> in cluster <EBI>.
Job was executed on host(s) <ebi5-188>, in queue <production-rh6>, as user <fg_atlas> in cluster <EBI>.
</net/isilonP/public/rw/homes/fg_atlas> was used as the home directory.
</nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827> was used as the working directory.
Started at Fri Mar 18 15:15:58 2016
Results reported at Fri Mar 18 15:18:55 2016

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
/nfs/ma/home/atlas3-production/sw/atlasinstall_prod/atlasprod/analysis/run_coexpression_for_experiment.R E-MTAB-3827 /nfs/ma/home/atlas3-production/analysis/baseline/rna-seq/experiments/E-MTAB-3827  
------------------------------------------------------------

Successfully completed.

Resource usage summary:

    CPU time :               155.79 sec.
    Max Memory :             659 MB
    Average Memory :         152.00 MB
    Total Requested Memory : 80000.00 MB
    Delta Memory :           79341.00 MB
    (Delta: the difference between total requested memory and actual max usage.)
    Max Swap :               2954 MB

    Max Processes :          13
    Max Threads :            14

The output (if any) is above this job summary.



PS:

Read file <coexpression.err> for stderr output of this job.

